lxd/db/openfga/openfga.go

```diff
@@ -484,12 +616,15 @@ WHERE auth_groups_permissions.entitlement = ? AND auth_groups_permissions.entity
 		})
 	}
 
+	// Cache the tuples.
+	cache.WriteReadStartingFromUserCache(cacheKey, tuples)
+
 	return storage.NewStaticTupleIterator(tuples), nil
 }
 
 // ReadPage is not implemented. It is not required for the functionality we need.
-func (*openfgaStore) ReadPage(ctx context.Context, store string, tk *openfgav1.TupleKey, opts storage.ReadPageOptions) ([]*openfgav1.Tuple, []byte, error) {
-	return nil, nil, api.StatusErrorf(http.StatusNotImplemented, "not implemented")
+func (*openfgaStore) ReadPage(ctx context.Context, store string, tk *openfgav1.TupleKey, opts storage.ReadPageOptions) ([]*openfgav1.Tuple, string, error) {
+	return nil, "", api.StatusErrorf(http.StatusNotImplemented, "not implemented")
```

Comment:
These changes are related to the update to OpenFGA 1.8.0. Could you include the commit(s) from  https://github.com/canonical/lxd/pull/14440 instead?

Reply:
fixed